Public/Src/Cache/ContentStore/Library/Tracing/EvictResult.cs

```diff
@@ -22,6 +22,7 @@ public EvictResult(long evictedSize, long evictedFiles, long pinnedSize, DateTim
             LastAccessTime = lastAccessTime;
             SuccessfullyEvictedHash = successfullyEvictedHash;
             ReplicaCount = replicaCount;
+            Age = DateTime.UtcNow - LastAccessTime;

```

Comment:
>LastAccessTime [](start = 36, length = 14)

Perhaps add UTC in the name or add doc comment that this time should be in UTC...